module_mission_templates.py

```diff
@@ -221,14 +221,6 @@
     (server_add_message_to_log, "str_s1_has_left_the_game_with_id_reg0"),
 
     (call_script, "script_log_equipment", ":player_id"),
-
-    (try_begin),
-      (player_get_slot, ":freeze_instance_id", ":player_id", slot_player_freeze_instance_id),

```

Comment:
Need to delete this player slot from the header|_ file and probably from where it is assigned considering it now serves no purpose.